plugin/src/main/kotlin/com/vanniktech/maven/publish/MavenPublishBaseExtension.kt

```diff
@@ -153,11 +152,6 @@ abstract class MavenPublishBaseExtension(
       project.gradleSigning.sign(publication)
     }
 
-    // TODO: https://youtrack.jetbrains.com/issue/KT-46466 https://github.com/gradle/gradle/issues/26091
-    project.tasks.withType(AbstractPublishToMaven::class.java).configureEach { publishTask ->
-      publishTask.dependsOn(project.tasks.withType(Sign::class.java))
-    }
```

Comment:
This was just fixing the issue for the signing task but not for the publish and metadata tasks.